netsuite/10.0/modules/ROOT/pages/netsuite-reference.adoc

```diff
@@ -990,7 +990,7 @@ Used to change the email address for the account.
 |===
 | Name | Type | Description | Default Value | Required
 | Configuration | String | The name of the configuration to use. | | x
```

Comment:
No periods at the ends of fragments in this column:
```suggestion
| Configuration | String | Name of the configuration to use | | x
```
I removed "The" for parallelism; will do so below as well.